cdap-watchdog/pom.xml

```diff
@@ -31,6 +31,18 @@
   <name>CDAP Watchdog</name>
   <packaging>jar</packaging>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.opentelemetry</groupId>
+        <artifactId>opentelemetry-bom</artifactId>
```

Comment:
If you use this, you should be able to remove specific versions from your OTEL dependency list.